### PR TITLE
Describe process by which a contributor can become a core developer

### PR DESCRIPTION
**Pull request by hadley**
How does this process sound to everyone?

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -11,7 +11,9 @@ This project is led by a benevolent dictator and managed by a core team of devel
...
+Core developers are contributors who have made several valuable contributions to the project and are now relied upon to both write code directly to the repository and screen the contributions of others. Core developers are empowered to merge pull requests after careful review. Core developers have no authority over the overall direction of the project, however it is their job to help develop or elicit appropriate contributions to the project. As a matter of policy, core developers, even if no longer active, are listed (chronologicall) in [package authors](https://ggplot2.tidyverse.org/authors.html).
+
+An invitation to join the core team will be extended to contributors who either have made a major contribution as a contributor, for example, they have added one major new feature or have a pattern of smaller contributions over an extended period of time. Any existing core developer can propose a contributor be invited to the core team. This decision is subject to confirmation by all existing authors (heavily weighted towards invitation, we only want to make sure we haven't missed some reason to not invite).
```
Comment (clauswilke) [inline GOVERNANCE.md]:
```suggestion
An invitation to join the core team will be extended to contributors who either have made a major contribution as a contributor, for example, they have added one major new feature, or have a pattern of smaller contributions over an extended period of time. Any existing core developer can propose a contributor to be invited to the core team. This decision is subject to confirmation by all existing authors (heavily weighted towards invitation, we only want to make sure we haven't missed some reason to not invite). 
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ This project is led by a benevolent dictator and managed by a core team of devel
...
+Core developers are contributors who have made several valuable contributions to the project and are now relied upon to both write code directly to the repository and screen the contributions of others. Core developers are empowered to merge pull requests after careful review. Core developers have no authority over the overall direction of the project, however it is their job to help develop or elicit appropriate contributions to the project. As a matter of policy, core developers, even if no longer active, are listed (chronologically) in [package authors](https://ggplot2.tidyverse.org/authors.html).
+
+An invitation to join the core team will be extended to contributors who either have made a major contribution as a contributor, for example, they have added one major new feature, or have a pattern of smaller contributions over an extended period of time. Any existing core developer can propose a contributor to be invited to the core team. This decision is subject to confirmation by all existing authors (heavily weighted towards invitation, we only want to make sure we haven't missed some reason to not invite).
```
[the remark this answers is not in the document]
Comment (hadley) [reply GOVERNANCE.md]: Maybe I should just say something like "non-response will be taken as a vote in favour".  But that implies it's a voting system, which I don't like either.

[the remark this answers is not in the document]
Comment (batpigandme) [reply GOVERNANCE.md]: How about taking out "all"? ⇨ "This decision is subject to confirmation by existing authors."

**Comment by hadley** [comment]
I did some more tweaking — I'm going to merge now so we have the updated policy and I'll try take a deeper look at the whole doc in the near future.

**Comment by kohske** [comment]
LGTM :)

Sorry for the belated reply.

**Comment by wch** [comment]
I know I'm a little late to the party, but I'm wondering about this part:

> Core developers have no authority over the overall direction of the project, however it is their job to help develop or elicit appropriate contributions to the project.

I think it would be clearer and more accurate if it said:

> Core developers have no authority over the overall direction of the project; their role is to help develop or elicit appropriate contributions to the project.

If it's my *job* to do those things, I'm not doing well at it. If it's my *role*, then I'm doing OK. :)

**Comment by hadley** [comment]
@wch this got heavily revised in https://github.com/tidyverse/ggplot2/pull/2961

**Pull request merged**